src/types/IUiaData.h

```diff
@@ -36,6 +36,8 @@ namespace Microsoft::Console::Types
         virtual const bool IsSelectionActive() const = 0;
         virtual void ClearSelection() = 0;
         virtual void SelectNewRegion(const COORD coordStart, const COORD coordEnd) = 0;
+        virtual COORD GetSelectionAnchor() = 0;

```

Comment:
```suggestion
        virtual const COORD GetSelectionAnchor() const = 0;
```
I assume you don't want to allow people to modify the state. But I could be wrong.

Reply:
Yes, I don't want people to modify the state. Thanks for the advice.